src/components/common/Summary/Summary.js

```diff
@@ -0,0 +1,175 @@
+  },
+})(Tabs);
+
+const AntTab = withStyles(theme => ({
+  root: {
+    borderRight: '1px solid #b6b6b6',
+    borderBottom: '1px solid #b6b6b6',
+    textAlign: 'center',
+    textTransform: 'uppercase',
+    color: 'black',
+    minWidth: 72,
+    fontWeight: 700,
+    paddingRight: theme.spacing(4),
+    paddingLeft: theme.spacing(4),
+    fontFamily: [
+      '-apple-system',
+      'BlinkMacSystemFont',
+      '"Segoe UI"',
+      'Roboto',
+      '"Helvetica Neue"',
+      'Arial',
+      'sans-serif',
+      '"Apple Color Emoji"',
+      '"Segoe UI Emoji"',
+      '"Segoe UI Symbol"',
+    ].join(','),
+    '&$selected': {
+      color: '#d58e32',
+      fontWeight: 700,
+      borderBottom: '1px solid #000000',
+    },
+    '&:focus': {
+      color: '#d58e32',
+      borderBottom: 'none',
+    },
+  },
+  selected: {},
+}))(props => <Tab disableRipple {...props} />);
+
+const StyledTab = withStyles(theme => ({
+  root: {
+    textTransform: 'none',
+    color: '#fff',
+    fontWeight: theme.typography.fontWeightRegular,
+    fontSize: theme.typography.pxToRem(15),
+    marginRight: theme.spacing(1),
+    '&:focus': {
+      opacity: 1,
+    },
+  },
+}))(props => <Tab disableRipple {...props} />);
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    flexGrow: 1,
+  },
+  padding: {
+    padding: theme.spacing(3),
+  },
+  demo1: {
+    backgroundColor: theme.palette.background.paper,
+  },
+  demo2: {
+    backgroundColor: '#2e1534',
+  },
+}));
+
+export default function Summary({
+  tab1,
+  tab2,
+  tab3,
+  tab4,
+  inner1,
+  inner2,
+  inner3,
+  inner4,
+}) {
+  function TabPanel(props) {
+    const { children, value, index, ...other } = props;
+
+    return (
+      <div
+        role='tabpanel'
+        hidden={value !== index}
+        id={`simple-tabpanel-${index}`}
+        aria-labelledby={`simple-tab-${index}`}
+        {...other}
+      >
+        {value === index && (
+          <Box p={3}>
+            <Typography>{children}</Typography>
+          </Box>
+        )}
+      </div>
+    );
+  }
+
+  TabPanel.propTypes = {
+    children: PropTypes.node,
+    index: PropTypes.any.isRequired,
+    value: PropTypes.any.isRequired,
+  };
+
+  function a11yProps(index) {
+    return {
+      id: `simple-tab-${index}`,
+      'aria-controls': `simple-tabpanel-${index}`,
+    };
+  }
+
+  const useStyles = makeStyles(theme => ({
+    root: {
+      flexGrow: 1,
+      backgroundColor: theme.palette.background.paper,
+    },
+  }));
+
+  const classes = useStyles();
+  const [value, setValue] = React.useState(0);
+
+  const handleChange = (event, newValue) => {
+    setValue(newValue);
+  };
+
+  return (
+    <div className={classes.root}>
+      <div className={classes.demo1}>
+        <AntTabs value={value} onChange={handleChange} aria-label='ant example'>
+          <AntTab label={tab1} />
+          <AntTab label={tab2} />
+          <AntTab label={tab3} />
+          <AntTab label={tab4} />
```

Comment:
I tu powinna być pętla

Reply:
a jak wrzuce potem sekcje komentarzy do jednego z tabow?

Reply:
napisałem w kolejnych uwagach jak to ogarnąć :)